frontend/public/style.css

```diff
@@ -5,6 +5,7 @@
   --primary-s: 84%;
   --primary-l: 52%;
   --primary: hsl(var(--primary-h) var(--primary-s) var(--primary-l));
+  --primary-highlight: #EB991D;
```

Comment:
Hold on, shouldn't this colour just be the `--primary`? It's just used for the state stroke when selected right?

Reply:
Haha yep, my bad. I kinda just assumed it was a different colour. I'll swap it over